linter.py

```diff
@@ -464,9 +468,16 @@ def linter_check(repo_root, linter_subfolder):
 
         if not (cpp_lint_success and pylint_success):
             print("=" * 80)
-            print("Commit rejected! Please address the linter errors above.")
+            print("Commit not up to standards!")
+            print("Please address the linter errors above.")
+            print("All of these linter errors must be resolved before merge.")
```

Comment:
One more thing, I think this last message is only relevant for a non-blocking linter. In a blocking linter, they must even be resolved before the commit.